code/datums/ai_laws.dm

```diff
@@ -29,7 +29,7 @@
 /datum/ai_laws/default/asimov
 	name = "Three Laws of Robotics"
 	id = "asimov"
-	inherent = list("You may not injure a player or, through inaction, allow a human being to come to harm.",\
+	inherent = list("You may not injure a player or, through inaction, allow a player to come to harm.",\

```

Comment:
```suggestion
	inherent = list("You may not injure a player and allow a player to come to harm.",\
```

Reply:
no nono, dennok is very wrong here, he is ESL

Reply:
all fixed, no need to get spicy here gamers